tests/countries/test_micronesia.py

```
@@ -0,0 +1,533 @@
+            "2023-12-25",
+        )
+        for subdiv, holidays in self.subdiv_holidays.items():
+            if subdiv == "YAP":
+                self.assertHolidayName(
+                    name, holidays, (f"{year}-12-24" for year in range(1987, 2050))
+                )
+                self.assertHolidayName(f"{name} (observed)", holidays, dt)
+            else:
+                self.assertNoHolidayName(name, holidays)
+
+    def test_2025(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)